src/test/java/pl/gov/coi/pomocua/ads/accomodations/AccommodationsResourceTest.java

```diff
@@ -1,15 +1,57 @@
 package pl.gov.coi.pomocua.ads.accomodations;
 
+import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
 import pl.gov.coi.pomocua.ads.BaseResourceTest;
 import pl.gov.coi.pomocua.ads.PageableResponse;
 
 import java.util.List;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 class AccommodationsResourceTest extends BaseResourceTest<AccommodationOffer> {
 
+    @Test
+    void shouldReturnOffersByFullCriteria() {
```

Comment:
How about a test that filters out by criteria. Check that the filtering is actually there

Reply:
done